tsconfig.json

```diff
@@ -11,5 +11,5 @@
     "types": ["node"]
   },
   "include": ["src"],
-  "exclude": []
+  "exclude": ["**/__tests__"]
```

Comment:
do we need to exclude tests?

Reply:
Yes, otherwise `tsc` will also compile `src/__tests__/**` to `build/`